pkg/cmd/pr/create/create_test.go

```diff
@@ -1225,6 +1231,127 @@ func Test_determineTrackingBranch(t *testing.T) {
 	}
 }
 
+func Test_draft(t *testing.T) {
```

Comment:
It duplicates `Test_createRun` https://github.com/cli/cli/blob/e545c7dee1d2fd105f26e3847139906bf4e1e886/pkg/cmd/pr/create/create_test.go#L257C1-L258C1

since it includes `expectedOutputs` and the assertion `test.ExpectLines(t, output.String(), tt.expectedOutputs...)`

Probably this could be refactored and changed in `Test_createRun`

What do you think?

Reply:
Initially, refactoring `Test_createRun` makes sense.  One thing that did come up while looking at `test.ExpectLines` was https://github.com/cli/cli/pull/2803 which I don't think is a problem for what you're doing as we have exact matches rather than regexs

Reply:
> Initially, refactoring Test_createRun makes sense

See https://github.com/cli/cli/pull/8376/commits/e1f6218148ec9c833e3a88a1c61aa11b549eb576 

> One thing that did come up while looking at test.ExpectLines was https://github.com/cli/cli/pull/2803 which I don't think is a problem for what you're doing as we have exact matches rather than regexs

See da237bd4